m4/acltx_prog_kpsewhich.m4

```diff
@@ -0,0 +1,20 @@
+dnl @synopsis ACLTX_PROG_KPSEWHICH([ACTION-IF-NOT-FOUND])
+dnl
+dnl This macro find a kpsewhich application and set the variable
+dnl kpsewhich to the name of the application or to no if not found if
+dnl ACTION-IF-NOT-FOUND is not specified, configure fail when then
+dnl application is not found.
```

Comment:
This whole paragraph is very strangely worded. Perhaps it should be something more like:
```
This macro looks for the kpsewhich application and sets the variable
kpsewhich to the name of the application if found or no if not found. If
ACTION-IF-NOT-FOUND is not specified when application is not found then fail.
```